ChangeLog.txt

```diff
@@ -16,6 +16,7 @@ Minicraft+ version 2.2.0
 * Changed a better logging
 * Fixed seed random problem
 * Improved the localization support
+* Changed how the game uses the --savedir command line argument. Now, when specified, the game will use it as the final game directory path instead of adding an extra '.playminicraft/mods/Minicraft_Plus' to it.
```

Comment:
You should probably rewrite this to make it shorter and simpler like the entries above it.